data/_ontologies/Body parts.csv

```diff
@@ -0,0 +1,119 @@
+order,name,label,parent,codesystem,code,ontologyTermURI,definition
```

Comment:
ah, dat is ook heel fijn. David en Willemijn hadden het hier ook al over want op individuele samples nivo ook nodig.